R/evaluate-clusters.R

```diff
@@ -7,18 +7,26 @@
 #' @param x Either a matrix of principal components (PCs), or a SingleCellExperiment
 #'  or Seurat object containing PCs. If a matrix is provided, rows should be cells
 #'  and columns should be PCs, and row names should be cell ids (e.g., barcodes).
-#' @param cluster_df A data frame that contains at least the columns `cell_id` and
-#'  `cluster`. The `cell_id` values should match either the PC matrix row names,
-#'  or the SingleCellExperiment/Seurat object cell ids. Typically this will be output from
-#'  the `rOpenScPCA::calculate_clusters()` function.
+#' @param cluster_df A data frame that contains at least two columns: one representing
+#'   unique cell ids, and one containing cluster assignments. By default, these columns
+#'   should be named `cell_id` and `cluster` respectively, though this can be customized.
+#'   The cell id column's values should match either the PC matrix row names, or the
+#'   SingleCellExperiment/Seurat object cell ids. Typically this data frame will be
+#'   output from the `rOpenScPCA::calculate_clusters()` function.
+#' @param cluster_col The name of the column in `cluster_df` which contains cluster
+#'   assignments. Default value is "cluster".
+#' @param cell_id_col The name of the column in `cluster_df` which contains unique cell
+#'   ids "cell_id" is assumed by default.
```

Comment:
I meant to get rid of all of the "assumed", though I didn't comment on all of them.
```suggestion
#'   ids. Default value is "cell_id".
```